core/middleware/middleware.go

```diff
@@ -103,27 +96,11 @@ func (this *Middleware) SetBinary(binary string) error {
 		this.Binary = ""
 		return nil
 	}
-	testCommand := exec.Command(binary)
```

Comment:
I believe this is the duplicated middleware checking you mentioned. If we remove it, does it mean using `hoverfly` command to set middleware would have no checking done?

Reply:
There is still checking done, this was the preliminary checks.

A test execute happens with the new middleware here:

https://github.com/SpectoLabs/hoverfly/pull/706/files/faaa6d794102efcf405bd1493d9d2827a9287d24#diff-e5ffef0351b641e124a65ce90d9017b4R161